packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -32,6 +32,13 @@
         }
     }
 
+    .p2p-my-ads__form-field--focused {
```

Comment:
Fixed rate field is not wrapped under `dc-input__wrapper`, thus declaring the new subclass outside `dc-input__wrapper` :
![Screenshot 2022-08-24 at 1 47 17 PM](https://user-images.githubusercontent.com/103016120/186343522-2ff6cb7b-07f2-4265-a2d8-18b97bf8ab76.png)

Reply:
can we use `label_className` prop?

Reply:
I didn't knew there was `label_className` 👀 , thanks for the tip!